src/health/index.ts

```diff
@@ -2,12 +2,15 @@ import { RPCServiceClient } from "@textile/grpc-powergate-client/dist/health/rpc
 import { Config, healthTypes } from "../types"
 import { promise } from "../util"
 
+export interface Health {
```

Comment:
unfortunately that meant i needed the interfaces for all the subsections documented